Fix "cast from pointer to integer of different size" warning when sizeof(void*) != sizeof(uint64_t)
--- xf86-video-qxl-0.1.1/src/spiceqxl_display.c.orig	2013-10-20 16:53:29.000000000 +0200
+++ xf86-video-qxl-0.1.1/src/spiceqxl_display.c	2013-10-20 21:46:26.895791164 +0200
@@ -350,6 +350,6 @@
 
 void spiceqxl_display_monitors_config(qxl_screen_t *qxl)
 {
-    spice_qxl_monitors_config_async(&qxl->display_sin, (QXLPHYSICAL)qxl->monitors_config,
+    spice_qxl_monitors_config_async(&qxl->display_sin, (QXLPHYSICAL)(uintptr_t)qxl->monitors_config,
                                     MEMSLOT_GROUP, 0);
 }
